MapboxDirections/MBRoadClass.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+@objc(MBRoadClass)
+public enum RoadClass: Int, CustomStringConvertible {
+    case toll
+    
+    case restricted
+    
+    case highway
+    
+    case ferry
+    
+    public init?(description: String) {
+        let level: RoadClass
+        switch description {
+        case "toll":
+            level = .toll
+        case "restricted":
+            level = .restricted
+        case "highway":
```

Comment:
Apparently the Directions API will use a `motorway` class, not a `highway` class.